### PR TITLE
Product listing page

### PR DESCRIPTION
**Pull request by rahulyadav139**
product listing page is created

### CODE REVIEW

**Review by j24m — APPROVED** [review]

File: products.css [hunk 1]
```diff
@@ -0,0 +1,87 @@
...
+}
+.listing-section__sorting {
+  margin-bottom: 1rem;
+}
+.card.ecom {
+  width: 100%;
+  cursor: pointer;
+}
+
+.listing-section__pages {
```
Comment (tarunsankhla) [inline products.css:22]: follow any one type of classname naming dash or snake I guess Vikram told in sesion

File: app.js [hunk 2]
```diff
@@ -1,47 +1,49 @@
-const slides = document.querySelectorAll('.slide');
-const btnLeft = document.querySelector('.btn-left');
-const btnRight = document.querySelector('.btn-right');
+// const slides = document.querySelectorAll('.slide');
```
Comment (tarunsankhla) [inline app.js:1]: The UI is looking great add a little gap/padding on filter checkbox list

**Review by tarunsankhla — APPROVED** [review]

**Review by j24m — APPROVED** [review]
Your code looks fine to me, And UI is even more amazing than that, just one thing please make small PRs , It gets a bit hectic for the reviewer to check so many lines of code and so many files at the same time. 
Just one change that I can think of in the UI, a bit of line height or margin between the checkbox category.

File: app.js (hunk 2 above)
Comment (rahulyadav139) [reply app.js:1]: I'll check definitely if it is needed then I will make that happen.

File: products.css (hunk 1 above)
Comment (rahulyadav139) [reply products.css:22]: I'll keep that in mind. Thank you for the review.

**Comment by rahulyadav139** [comment]
> Your code looks fine to me, And UI is even more amazing than that, just one thing please make small PRs , It gets a bit hectic for the reviewer to check so many lines of code and so many files at the same time. Just one change that I can think of in the UI, a bit of line height or margin between the checkbox category.

Thank you for the review. I'll make these changes.

**Pull request merged**